topics/variant-analysis/tutorials/exome-seq/tutorial.md

```diff
@@ -25,290 +34,902 @@ contributors:
+>      - *"Set of columns to include in the variant report table"*:
+>        `Custom (report user-specified columns)`
+>        - *"Choose columns to include in the report"*:
+>          - {% icon param-check %} *"alternative allele frequency (max_aaf_all)"*
+>        - *"Additional columns (comma-separated)"*:
+>          `chrom, start, ref, alt, impact, gene, clinvar_sig,
+>          clinvar_disease_name, clinvar_gene_phenotype, rs_ids`
+>
+>    > ### {% icon comment %} Variant- *vs* gene-centric ClinVar information
+>    > Most annotation data offered by GEMINI is variant-centric, *i.e.*,
+>    > annotations reported for a variant are specific to this exact variant.
+>    > A few annotation sources, however, also provide gene-centric
+>    > information, which applies to the gene affected by a variant, not
+>    > necessarily the variant itself.
+>    >
+>    > In the case of [ClinVar](https://www.ncbi.nlm.nih.gov/clinvar/), the
+>    > annotation fields/columns `clinvar_sig` and `clinvar_disease_name` refer
+>    > to the particular variant, but `clinvar_gene_phenotype` provides
+>    > information about the affected gene.
+>    >
+>    > Including the gene phenotype in the report can be crucial because a
+>    > gene may be well known to be disease-relevant, while a particular
+>    > variant may not have been clinically observed or been reported before.
+>    {: .comment}
+>
+{: .hands_on}
+
+> > ### {% icon details %} More GEMINI usage examples
 > >
-> > To go further on **Gemini**, you can have a look at the following tutorials:
+> > While only demonstrating command line use of GEMINI, the following tutorial
+> > slides may give you additional ideas for variant queries and filters:
 > >
-> > - [Introduction](https://s3.amazonaws.com/gemini-tutorials/Intro-To-Gemini.pdf)
-> > - [Identifying *de novo* mutations underlying Mendelian disease](https://s3.amazonaws.com/gemini-tutorials/Gemini-DeNovo-Tutorial.pdf)
-> > - [Identifying autosomal recessive variants underlying Mendelian disease](https://s3.amazonaws.com/gemini-tutorials/Gemini-Recessive-Tutorial.pdf)
-> > - [Identifying autosomal dominant variants underlying Mendelian disease](https://s3.amazonaws.com/gemini-tutorials/Gemini-Dominant-Tutorial.pdf)
+> > - [Introduction to GEMINI](https://s3.amazonaws.com/gemini-tutorials/Intro-To-Gemini.pdf)
+> > - [Identifying *de novo* mutations with GEMINI](https://s3.amazonaws.com/gemini-tutorials/Gemini-DeNovo-Tutorial.pdf)
+> > - [Identifying recessive gene candidates with GEMINI](https://s3.amazonaws.com/gemini-tutorials/Gemini-Recessive-Tutorial.pdf)
+> > - [Identifying dominant gene candidates with GEMINI](https://s3.amazonaws.com/gemini-tutorials/Gemini-Dominant-Tutorial.pdf)
 > {: .details}
-> And for a more detailed tutorial on variant data generation in Galaxy, have a
-> look at the [tutorial on diploid variant calling](https://github.com/nekrut/galaxy/wiki/Diploid-variant-calling).
-{: .comment}
+
```

Comment:
Is there any interesting results in the output? Could you add a question box or something to inspect and think about the results and relate to the original question? Thanks